docs/source/backends/raspbian.rst

```diff
@@ -0,0 +1,683 @@
+================
+Raspbian Backend
+================
+
+.. include:: ../_github.rst
+
+The ``Raspbian`` backend allows to Raspbian compatible configuration files.
+
+Initialization
+--------------
+
+.. automethod:: netjsonconfig.Raspbian.__init__
+
+
+Render method
+-------------
+
+.. automethod:: netjsonconfig.OpenWrt.render
+
+Code example:
+
+.. code-block:: python
+
+    from netjsonconfig import OpenWrt
+
+    o = Raspbian({
+        "interfaces": [
+            {
+                "name": "eth0.1",
+                "type": "ethernet",
+                "addresses": [
+                    {
+                        "address": "192.168.1.1",
+                        "mask": 24,
+                        "proto": "static",
+                        "family": "ipv4"
+                    },
+                    {
+                        "address": "192.168.2.1",
+                        "mask": 24,
+                        "proto": "static",
+                        "family": "ipv4"
+                    },
+                    {
+                        "address": "fd87::1",
+                        "mask": 128,
+                        "proto": "static",
+                        "family": "ipv6"
+                    }
+                ]
+            }
+        ]
+    })
+    print o.render()
+
+Will return the following output::
+
+    config: /etc/network/interfaces
+    auto eth0.1
+    iface eth0.1 inet static
+      address 192.168.1.1
+      netmask 255.255.255.0
+    iface eth0.1 inet static
+      address 192.168.2.1
+      netmask 255.255.255.0
+    iface eth0.1 inet6 static
+      address fd87::1
+      netmask 128
+
+Generate method
+---------------
+
+.. automethod:: netjsonconfig.Raspbian.generate
+
+Code example:
+
+.. code-block:: python
+
+    >>> import tarfile
+    >>> from netjsonconfig import Raspbian
+    >>>
+    >>> o = Raspbian({
+    ...    "interfaces": [
+    ...      {
```

Comment:
use 4 spaces for indentation for consistency with the rest of the project please